maxim/tests/test_llamaindex.py

```
@@ -0,0 +1,257 @@
+import unittest
+import os
+import dotenv
+import asyncio
+from typing import Dict, Optional
```

Ruff (0.12.2)

`typing.Dict` is deprecated, use `dict` instead

(UP035)